rocAL/rocAL/include/meta_data_reader.h

```diff
@@ -71,6 +75,9 @@ struct MetaDataConfig
     unsigned sequence_length() const { return _sequence_length; }
     unsigned frame_step() const { return _frame_step; }
     unsigned frame_stride() const { return _frame_stride; }
+    bool keypoint() const { return _keypoint; }
```

Comment:
Where this is used?

Reply:
It is not needed  will remove it. It is not needed since now we are using a separate  reader for keypoints